source/_components/switch.telnet.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Telnet Switch"
+description: "Instructions how to integrate telnet switches into Home Assistant."
```

Comment:
Telnet is commonly written with a capital T in documentation. This isn't necessarily correct but fairly common.